### DISCUSSION TITLE
arch 6.3.2 - plasma 5.27.5

### DISCUSSION THREAD

**Discussion by lomik31** [ffcrus/garlic#3] [General]
Оба скрипта прекрасно работают на Arch. Звук не тестил.

Небольшое дополнение:
Использую программу для сохранения буфера обмена — copyq.
И проблема была в том, что из-за использования буфера как хранилища, в истории это было видно, тем самым засоряя её (историю).
Решил проблему добавлением строчки `copyq remove 0` после `xsel --clear` в оба скрипта.
После этого живу в счастье.

Кстати, хотел бы еще одну вариацию: изменяется не вся последняя строка, а только последнее слово, если после него не стоит пробел

**Comment by evpix** [comment]
> Решил проблему добавлением строчки `copyq remove 0` после `xsel --clear` в оба скрипта.
> После этого живу в счастье.

Спасибо за подсказку. Реально помогло.
